examples/robot.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+from typing import List
+
+from imm.pybullet_util.typing_extra import TranslationT, QuaternionT
+from imm.pybullet_util.bullet_client import BulletClient
+from imm.pybullet_util.common import get_joint_limits
+
+
+class UR5RG2(object):
+    def __init__(self, bc: BulletClient, pos: TranslationT, orn: QuaternionT):
+        self.bc = bc
+        self.robot_id: int = bc.loadURDF('./urdf/ur5_rg2.urdf', pos, orn,
+                                         useFixedBase=True, flags=bc.URDF_USE_SELF_COLLISION_EXCLUDE_PARENT)
+        # Camera setup
+        self.projectionMatrix = bc.computeProjectionMatrixFOV(fov=58, aspect=1.5, nearVal=0.28, farVal=5)
+        self.image_renderer = bc.ER_BULLET_HARDWARE_OPENGL
+
+        # Disable closed linkage collision (gripper's inner fingers)
+        bc.setCollisionFilterPair(self.robot_id, self.robot_id, 9, 12, 0)
+        bc.setCollisionFilterPair(self.robot_id, self.robot_id, 9, 15, 0)
+
+        self.ee_id: int = 7  # NOTE(choi): End-effector joint ID for UR5 robot
+        self.arm_joint_ids: List[int, ...] = [1, 2, 3, 4, 5, 6]  # NOTE(choi): Hardcoded arm joint id for UR5 robot
+        self.arm_joint_limits = get_joint_limits(bc, self.robot_id, self.arm_joint_ids)
+
+        self.arm_rest_poses: List[float, ...] = [0, -1.5446774605904932, 1.343946009733127, -1.3708613585093699,
+                                                 -1.5707970583733368, 0]  # NOTE(choi): Manually set arm rest poses
+
+        self.gripper_link_ids: List[int, ...] = [10, 11, 12, 13, 14, 15]
+        self.gripper_link_multiplier: List[float, ...] = [-1, -1, 1, 1, 1, -1]
+        self.gripper_link_limit: List[float, float] = [0, 1.343904]
```

Comment:
It can seem strange but `List[float,float]` isn't valid...
```suggestion
        self.gripper_link_limit: Tuple[float, float] = (0, 1.343904)
```